test/gtest/ucs/test_bitmap.cc

```diff
@@ -368,3 +384,105 @@ UCS_TEST_F(test_dynamic_bitmap, test_get_set_reset)
 
     EXPECT_TRUE(ucs_dynamic_bitmap_is_zero(&m_bitmap));
 }
+
```

Comment:
maybe we need to add tests for at least some of the `ucs_bitmap_bits_*()` functions?

Reply:
they should all be tested by static_bitmap/dynamic_bitmap, since they are internal helper function IMO they don't deserve a separate testing..